bootloader/bootloader.go

```diff
@@ -139,6 +139,12 @@ type ExtractedRunKernelImageBootloader interface {
 	DisableTryKernel() error
 }
 
+// ManagedBootloader can manage its own boot assets (typically boot config).
+type ManagedBootloader interface {
+	// IsManaged returns true when the on disk boot assets are managed.
+	IsManaged() (bool, error)
```

Comment:
for a second I wonder why the error, then I see this is a dynamic query about what is on disk? so maybe IsCurrentlyManaged ?